src/game/mod.rs

```diff
@@ -22,13 +22,15 @@ use bevy_auto_plugin::auto_plugin::*;
 use bevy_egui::EguiPlugin;
 #[cfg(feature = "dev_frame_count_log")]
 use bevy_frame_count_log_prefix::prelude::FrameCountLogPrefixPlugin;
+use bevy_hanabi::HanabiPlugin;
 
 pub struct GamePlugin;
 
 impl Plugin for GamePlugin {
     #[auto_plugin(app=app)]
     fn build(&self, app: &mut App) {
         app.add_plugins(default_plugins());
+        app.add_plugins(HanabiPlugin);
```

Comment:
Another nitpick but we should probably have it loaded further down? I try to aim for order of importance

Reply:
Besides bevy default plugins how do you determine importance? Maybe we should do three categories - bevy, external, internal - and lexicographic sort within those?